Languages/IronPython/IronPython/Lib/iptest/process_util.py

```diff
@@ -148,6 +151,8 @@ def kill_process(arg):
 
 def get_clr_dir():
     import clr
-    from System import Type
-    from System.IO import Path
-    return Path.GetDirectoryName(Type.GetType('System.Int32').Assembly.Location)
+    import System
+    if is_netstandard:
+        clr.AddReference("System.Runtime.Extensions")
+        return System.IO.Path.Combine(System.Environment.GetEnvironmentVariable("windir"), r"Microsoft.NET\Framework\v4.0.30319")
```

Comment:
True, but as far as I know `get_clr_dir` is only used by `run_csc`, `run_ilasm` and `run_vbc` (and the first two don't call it on `posix`).